pkg/util/informermanager/interface.go

```diff
@@ -44,6 +44,17 @@ type EventHandlerGenerator struct {
 // When a FTC deletion is observed, latest will be nil.
 type FTCUpdateHandler func(lastObserved, latest *fedcorev1a1.FederatedTypeConfig)
 
+type FederatedTypeConfigManager interface {
+	// Returns the known FTC mapping for the given GVK if it exists.
+	GetResourceFTC(gvk schema.GroupVersionKind) (ftc *fedcorev1a1.FederatedTypeConfig, exists bool)
+
+	// Returns the FederatedTypeConfig lister used by the manager.
+	GetFederatedTypeConfigLister() fedcorev1a1listers.FederatedTypeConfigLister
```

Comment:
We should probably move `AddFTCUpdateHandler` here as well.

Reply:
I think if `GetFederatedTypeConfigLister` is unused, we can remove this method from both `FederatedTypeConfigManager` and `InformerManager`.

The underlying FTC lister might not match the `InformerManager`'s view of FTCs as the `InformerManager`'s view is still subject to its own internal processing and tends to lag behind.

cc @gary-lgy

Reply:
Will handle this in a subsequent pr.